File: mario/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height = get_int("Height: ");
    while (height <= 0)
    {
        height = get_int("Height: ");
    }

    for (int i = 1; i <= height; i++)
    {
        for (int j = 0; j < height - i; j++)
        {
            printf(" ");
        }
        for (int k = 0; k < i; k++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: sentimental-mario-more/mario.py
from cs50 import get_int

# Prompt the user for the pyramid's height
while True:
    height = get_int("Height: ")
    if 1 <= height <= 8:
        break  # Exit the loop if the input is valid

# Generate the half-pyramids
for i in range(1, height + 1):
    # Print the left pyramid with right alignment
    print(" " * (height - i) + "#" * i, end="")
    # Print the gap and the right pyramid
    print("  " + "#" * i)
